python/__init__.py

```diff
@@ -336,7 +341,7 @@ def get_architecture_descriptor(self, **kwargs):
 
     def make_launcher_stub(self, name, signature, constants, ids):
         # name of files that are cached
-        so_cache_key = make_so_cache_key(version_key(), signature, constants, ids)
+        so_cache_key = make_so_cache_key(self.version_key, signature, constants, ids)
```

Comment:
this should be `self.get_version_key()` because `self.version_key` is initially `None`